macos.md

```diff
@@ -24,11 +24,17 @@
    <img src="./macos-4.1-homebrew-next-steps.png"><br><br>
 6. Copy each line in the following text, paste it in the terminal and hit return.<br><br>
    ```bash
-   brew install flyctl git node@18 python yarn
+   brew install flyctl git node@18 python
    brew link --overwrite node@18
    ```
-   This uses Homebrew to install `flyctl`, Git, Node.js, Python and Yarn.<br><br>
+   This uses Homebrew to install `flyctl`, Git, Node.js, Python.<br><br>
 7. Copy each line in the following text, paste it in the terminal and hit return.<br><br>
+   ```bash
+   brew install corepack
```

Comment:
I don't think you need this - I tested it out without it and it worked for me

Reply:
I tested it without, and it also worked for me. The first time I added PNPM, I used the installation guide in the [PNPM documentation](https://pnpm.io/installation#using-corepack)
Commit: [297d87c](https://github.com/upleveled/system-setup/pull/24/commits/297d87c25ac2b035a4a6c7107baa0da8faa4844c)

Reply:
yeah I let Zoltan know about this a while back: https://github.com/pnpm/pnpm.github.io/issues/359#issuecomment-1469836101